migration-roadmap/thumbnail.md

```diff
@@ -0,0 +1,283 @@
+            src="[imageURL]"
+            alt="[altText]"
+        />
+    </div>
+</div>
+
+<!-- Default -->
+<div
+    class="spectrum-Thumbnail spectrum-Thumbnail--cover spectrum-Thumbnail-layer is-selected is-disabled is-focused spectrum-Thumbnail--size500"
+>
+    <div
+        class="spectrum-Thumbnail-background"
+        style="background-color: [backgroundColor]"
+    ></div>
+    <div class="spectrum-Thumbnail-image-wrapper">
+        <img
+            class="spectrum-Thumbnail-image"
+            src="[imageURL]"
+            alt="[altText]"
+        />
+    </div>
+</div>
+```
+
+</details>
+
+<details>
+<summary>Diff: Legacy (CSS main) → Spectrum 2 (CSS spectrum-two)</summary>
+
+No significant structural changes.
+
+</details>
+
+### CSS => SWC mapping
+
+| CSS selector                        | Attribute or slot      | Status          |
+| ----------------------------------- | ---------------------- | --------------- |
+| `.spectrum-Thumbnail--size50`       | `size="50"`            | Implemented     |
+| `.spectrum-Thumbnail--size75`       | `size="75"`            | Implemented     |
+| `.spectrum-Thumbnail--size100`      | `size="100"`           | Implemented     |
+| `.spectrum-Thumbnail--size200`      | `size="200"`           | Implemented     |
+| `.spectrum-Thumbnail--size300`      | `size="300"`           | Implemented     |
+| `.spectrum-Thumbnail--size400`      | `size="400"`           | Implemented     |
+| `.spectrum-Thumbnail--size500`      | `size="500"`           | Implemented     |
+| `.spectrum-Thumbnail--size600`      | `size="600"`           | Implemented     |
+| `.spectrum-Thumbnail--size700`      | `size="700"`           | Implemented     |
+| `.spectrum-Thumbnail--size800`      | `size="800"`           | Implemented     |
+| `.spectrum-Thumbnail--size900`      | `size="900"`           | Implemented     |
+| `.spectrum-Thumbnail--size1000`     | `size="1000"`          | Implemented     |
+| `.spectrum-Thumbnail--cover`        | `cover` attribute      | Implemented     |
+| `.spectrum-Thumbnail-layer`         | `layer` attribute      | Implemented     |
+| `.spectrum-Thumbnail-background`    | `background` attribute | Implemented     |
+| `.spectrum-Thumbnail-image`         | Default slot           | Implemented     |
+| `.spectrum-Thumbnail-image-wrapper` | Internal wrapper       | Implemented     |
+| `.spectrum-Thumbnail-layer-inner`   | Internal wrapper       | Implemented     |
+| `.spectrum-Thumbnail.is-disabled`   | `disabled` attribute   | Missing from WC |
+| `.spectrum-Thumbnail.is-focused`    | Focus state            | Missing from WC |
+| `.spectrum-Thumbnail.is-selected`   | `selected` attribute   | Missing from WC |
+| `.spectrum-Thumbnail:before`        | Pseudo-element styling | Missing from WC |
+| `.spectrum-Thumbnail:after`         | Pseudo-element styling | Missing from WC |
+| `.spectrum-Thumbnail-layer:before`  | Pseudo-element styling | Missing from WC |
```

Comment:
Let's take another look at these before we label them as potential gaps or new features that SWC needs to add, I know for sure that [disabled does appear to exist in SWC](https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/thumbnail--disabled&globals=system:spectrum-two), but some of the other states/pseudo-classes might actually be implemented. Might be worth comparing `spectrum-css/components/thumbnail/index.css` to `spectrum-web-components/packages/thumbnail/src/spectrum-thumbnail.css` and other SWC CSS files as well.